### PR TITLE
Details Page | Recipes App

### PR DESCRIPTION
**Pull request by richardwellerson**
- [x] Share and Favorite
- [x] Title and Subtitle
- [x] Image
- [x] Ingredients Div
- [x] Instructions Div
- [x] Embed Video
- [x] Recommended (Carousel Div)
- [x] Start Recipe Button

### CODE REVIEW

File: src/Components/FavoriteAndShare.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import Share from '../Images/Share.svg';
+import Favorite from '../Images/Favorite.svg';
+import Favorited from '../Images/Favorited.svg';
+import '../Styles/FavoriteAndShare.css';
+
+const heartClick = (isFavorite, setIsFavorite) => {
+  const localStorageUser = JSON.parse(localStorage.getItem('user'));
+  setIsFavorite(!isFavorite);
+  console.log(localStorageUser)
```
Comment (laurolyra) [inline src/Components/FavoriteAndShare.js:11]: maybe you should delete the line number 11.
Why not show an alert on hearclick?

File: src/Components/FavoriteAndShare.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+const FavoriteAndShare = () => {
+  const [copied, setCopied] = useState('');
+  const [isFavorite, setIsFavorite] = useState(false);
+  const message = () => {
+    setCopied('Copied!');
```
Comment (laurolyra) [inline src/Components/FavoriteAndShare.js:18]: Same here. Perhaps an lert would be useful

File: src/Components/FavoriteAndShare.js (hunk 1 above)
Comment (richardwellerson) [reply src/Components/FavoriteAndShare.js]: Because alert is too aggressive.

File: src/Components/FavoriteAndShare.js (hunk 2 above)
Comment (richardwellerson) [reply src/Components/FavoriteAndShare.js:65]: Alert is too aggressive to this app.

**Pull request merged**